test/cflat/branch/cflatp-branch.asm

```diff
@@ -0,0 +1 @@
+Compiled from "example.lir"
```

Comment:
We noticed that in this directory we needed to create a intermediate asm file between input and output file, but we do not know how to write the content. @maemre

Reply:
What do you mean by an intermediate assembly file? The compiler goes from LIR code to assembly, there is no intermediate output. If you need an expected output, you can generate it by running the compiler on the input file as usual.

Reply:
Since we took java-test as reference, for each test example, Java had an input file, an intermediate asm file, and a output asm file. So we thought we needed one also? Could you take a look at the Java's test folder and tell us why they need this intermediate asm file? We are confused by the usage of these files as well.
(We had already run the compiler on the input file and got the expected output file.)

Reply:
In case of Java, those files are there because of the particulars of the compilation steps the java compiler implementation uses, and the `processAsm` function is specialized for Java in that case. Those files aren't intermediate files, but they seem like the output files for each class (because Java generates each `.class` file separately). You can see that from other Java tests. You don't need such mechanisms for Cflat because there is one output file. All you need is an expected output file, and calling the compiler will generate the actual output which should be (near-)identical.